events/models.py

```diff
@@ -152,3 +152,22 @@ def save(self):
     
     def __unicode__(self):
         return self.name
+
+class Category(models.Model):
+    '''
+        Model for categories of events
+    '''
+
+    id = models.AutoField(primary_key=True)
+    descrip = models.CharField(_("Category"), max_length=100, default="")
```

Comment:
Can we use a full name for this? description

Also, maybe we want a name and description for each category, as well as a slug for linking it?